specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2019-08-01-preview/storagecache.json

```diff
@@ -526,11 +523,9 @@
         "responses": {
           "200": {
             "description": "All cache data is clean.",
-            "schema": {}
```

Comment:
isn't there a schema for this POST response?

Reply:
I think it's not needed if no response data expected. empty schema here is not valid and will be complained by autorest. Do you know how to include service team to validate the change? thanks.